src/ds++/DracoStrings.hh

```diff
@@ -115,6 +115,9 @@ template <typename T> auto parse_number_impl(std::string const &str) -> T;
 // specializations for these types are defined in DracoStrings.cc
 template <> auto parse_number_impl<int32_t>(std::string const &str) -> int32_t;
 template <> auto parse_number_impl<int64_t>(std::string const &str) -> int64_t;
+template <> auto parse_number_impl<long>(std::string const &str) -> long;
```

Comment:
Somehow I got into trouble not having these declarations. Trying to remember where they were needed; if I can't, will delete them.